### ISSUE TITLE
B: Generator uses conflicting variable names in config()

### ISSUE THREAD

**Issue by mk0a1a** [facebookincubator/ent#57]
When creating an entity that starts with "i", the index variable used in config conflicts with the struct variable.

For example:

1. Create entity
    ```bash
    entc init Item
    ```
2. Generate code
    ```bash
    entc generate ./ent/schema
    ```
3. Run code
    ```
    go run main.go
    ```
4. Result:
    ```bash
    ent/item.go:78:4: invalid operation: i[i] (type int does not support indexing)
    ```

Conflicting generated code:
```go
func (i Items) config(cfg config) {
	for i := range i {
		i[i].config = cfg
	}
}
```

"i" variable of type Items conflicts with "i" variable for index

**Comment by a8m** [comment]
Thanks for opening this issue. Sending a fix.

**Comment by a8m** [comment]
Closed via #58
